app/javascript/controllers/transcript_controller.js

```diff
@@ -68,8 +64,6 @@ export default class extends Controller {
   }
 
   setupTranscriptLanguageSwitching() {
-    if (this.captionTracks.length == 0) return
```

Comment:
`this.captionTracks` always returns an array, and we can safely `forEach` an empty array below, so I don't see a need for this check.